src/clients/scc.rb

```diff
@@ -37,25 +42,49 @@ def main
       textdomain "registration"
 
       if WFM.Args.include?("help")
-        cmdline_description = {
-          "id"   => "scc",
-          # Command line help text for the repository module, %1 is "SUSEconnect"
-          "help" => _("Use '%s' instead of this YaST module.") % "SUSEconnect"
-        }
-
-        CommandLine.Run(cmdline_description)
+        print_help
       else
         Wizard.CreateDialog
 
         begin
           ::Registration::SwMgmt.init
 
           return WFM.call("inst_scc", WFM.Args)
+        rescue Registration::SourceRestoreError => e
+          retry if fix_repositories(e.message)
         ensure
           Wizard.CloseDialog
         end
       end
     end
+
```

Comment:
nice, almost perfect, just please add here `private` so it is clear what is public API and what is internal helper.